### ISSUE TITLE
[SGE] Combo suggestion: Dosis 3 & Toxicon 2

### ISSUE THREAD

**Issue by oftheknife** [attickdoor/XIVComboPlugin#125]
**tl;dr combo: If stacks of sting, cast Toxicon2. If no stacks, cast Dosis3.** 

I believe the logical combo here is Dosis3 into Toxicon2: at level 90 they share damage and healing potency, and it would be efficient to be able to dump Addersting stacks before spamming Dosis in order to prevent overcapping on an otherwise useless resource. Consolidating these buttons would be a nice option. 

Another suggestion would be Toxicon/Dyskrasia on the same button - they are both technically AOE, but Dyskrasia is an Art of War style melee AOE and Toxicon has a 25y range, so it would be pretty awkward.

**Comment by mfaisca** [comment]
While I understand what you mean, you probably wouldn't want that since you want to save Addersting stacks for when you have to move and can't stand still to cast.

Combos are a toggle though, so I think just having the option would be neat

**Comment by Aelexe** [comment]
This would go against requirement 1 outlined in #119.
> The abilities in question must be truly mutually exclusive. If you can use A, there is literally no way to use B, and vice versa.

**Comment by attickdoor** [comment]
This makes absolutely no sense. Toxikon is a movement tool, you would never want to be forced to use it.

**Comment by oftheknife** [comment]
k
